juriscraper/opinions/united_states/state/nev.py

```diff
@@ -35,13 +52,48 @@ def _download(self):
 
     def _process_html(self):
         for case in self.html:
-            if "COA" in case["caseNumber"]:
+            if not self.correct_court(case):
                 continue
+
+            if (self.now - datetime.fromisoformat(case["date"])).days > 30:
+                # Stop at 30 days
+                break
             self.cases.append(
                 {
                     "name": case["caseTitle"],
                     "docket": case["caseNumber"],
                     "date": case["date"],
-                    "url": case["docurl"],
+                    "url": "placeholder",
                 }
             )
+
+    def fetch_document_link(self, csNumber: str):
```

Comment:
There is a lot of extra information on the case page, I think we could pick it up, since we are already downloading all the HTML

- At the top table, we can find lower_court and lower_court_number
- On the same row as the opinion link, the author_str can sometimes be picked up
- Also, in the same \<td> as the author_str,  there is a citation string like `139 Nev. Adv. Opn. No. 59`

![image](https://github.com/freelawproject/juriscraper/assets/14970769/8fd0107f-d20a-4286-ab34-9f15b94c94da)

Reply:
as per our conversation this is something we should address with an update to deferring list - because as it stands we dont have a smooth mechanism to do multiple attributes